src/client/ds/object_meta.h

```diff
@@ -131,6 +131,11 @@ class ObjectMeta {
    */
   bool const IsLocal() const;
 
+  /**
+   * @brief Whether the object meta and rpc client is located on the same node.
+   */
+  bool const IsLocated() const;
```

Comment:
Why we need it? Even the rpc client connects to the same node where the object live, it still a "remote" object and requires remote data access.

Reply:
Sorry about the confusing brief, actually it means whether the instance connected by rpc client is the same as meta's instance.

Reply:
Add a method to the RPCClient:

```
bool IsFetchable(const ObjectMeta &object_meta) { ... }
```